pydm/widgets/timeplot_curve_editor.py

```diff
@@ -0,0 +1,71 @@
+from ..PyQt.QtGui import QDialog, QVBoxLayout, QHBoxLayout, QTableView, QAbstractItemView, QSpacerItem, QSizePolicy, QDialogButtonBox, QPushButton, QItemSelection
+from ..PyQt.QtCore import Qt, pyqtSlot
```

Comment:
For me, it only works when `QItemSelection` is imported from `..PyQt.QtCore`. I mean:

`..PyQt.QtCore import Qt, pyqtSlot, QItemSelection`